packaging/deb/dotnet-sharedframework-debian_config.json

```diff
@@ -32,5 +32,9 @@
         "%HOSTFXR_DEBIAN_PACKAGE_NAME%" : {},
         "libssl1.0.0" : {},
         "libcurl3" : {}
-    }
+    },
+
+    "debian_ignored_dependencies" : [
+        "liblldb-3.6"
+    ]
```

Comment:
dh_helper which we're using to build the package auto detects package dependencies based libraries linked to the files inside the package. 

libsos....so  is linked to liblldb-3.6 and is only present in the sharedfx debian package.